aws/resource_aws_cloudsearch_domain.go

```diff
@@ -0,0 +1,486 @@
+package aws
```

Comment:
We should definitely add tests for this resource, so that it's easier to manage create, update, deletes, and check everything is fine in an automated way.
You can look at other resources if needed